usaspending_api/etl/tests/integration/test_load_transactions_in_delta.py

```diff
@@ -0,0 +1,1345 @@
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=True,
+        unique_award_key="award_procure_0002"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=7,
+        award=assist_awards[2],
+        transaction_unique_id="award_assist_0003_trans_0001",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=False,
+        unique_award_key="award_assist_0003"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=8,
+        award=assist_awards[2],
+        transaction_unique_id="award_assist_0003_trans_0002",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=False,
+        unique_award_key="award_assist_0003"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=9,
+        award=procure_awards[2],
+        transaction_unique_id="award_procure_0003_trans_0001",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=True,
+        unique_award_key="award_procure_0003"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=10,
+        award=procure_awards[2],
+        transaction_unique_id="award_procure_0003_trans_0002",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=True,
+        unique_award_key="award_procure_0003"
+    )
+
+    # `name` and `external_data_type_id` must match those in `usaspending.broker.lookups`
+    edt_tn = baker.make(
+        "broker.ExternalDataType",
+        name="transaction_normalized",
+        external_data_type_id=203,
+        update_date="2022-10-31"
+    )
+    baker.make("broker.ExternalDataLoadDate", last_load_date="2022-10-31", external_data_type=edt_tn)
+
+@fixture
+def finish_external_data_tables(populate_initial_awards_and_transaction_normalized_data):
+    # `name` and `external_data_type_id` must match those in `usaspending.broker.lookups`
+    edt_tidlu = baker.make(
+        "broker.ExternalDataType",
+        name="transaction_id_lookup",
+        external_data_type_id=205,
+        update_date=None
+    )
+    edt_aidlu = baker.make(
+        "broker.ExternalDataType",
+        name="award_id_lookup",
+        external_data_type_id=206,
+        update_date=None
+    )
+    baker.make("broker.ExternalDataLoadDate", last_load_date="1970-01-01", external_data_type=edt_tidlu)
+    baker.make("broker.ExternalDataLoadDate", last_load_date="1970-01-01", external_data_type=edt_aidlu)
+
+
+
+def load_initial_tables_to_delta_and_verify(spark, s3_data_bucket):
+    # Load tables and ensure they have loaded correctly
+    verify_delta_table_loaded_to_delta(spark, "published_fabs", s3_data_bucket)
+    verify_delta_table_loaded_to_delta(spark, "detached_award_procurement", s3_data_bucket)
+    verify_delta_table_loaded_to_delta(spark, "transaction_normalized", s3_data_bucket)
+    verify_delta_table_loaded_to_delta(spark, "awards", s3_data_bucket)
```

Comment:
I think it would make sense for a new function to be created that loads the table to delta without any sort of verification of the results. Especially since this is being run so many times throughout the tests

Reply:
I'm not totally sure why you would want this.  I guess we are verifying that the data in the Delta table matches what we expect, so I suppose the verification that the Delta table matches the Postgres table isn't strictly necessary.

If I create a new function to only load, but not verify the Delta table, should I handle the `alt_db` or `alt_name` arguments like the `verify_delta_table_loaded_to_delta` function in `test_load_to_from_delta.py` does?  If so, should I just split the function in `test_load_to_from_delta.py` into `load_delta_table_from_postgres` and `verify_delta_table_loaded_to_delta` functions?

Reply:
> I guess we are verifying that the data in the Delta table matches what we expect, so I suppose the verification that the Delta table matches the Postgres table isn't strictly necessary.

Yes, that's what I was thinking. Our tests are starting to take quite awhile to run, so anything we can do to cut back on runtime would help. In the current setup, we're essentially testing the same code multiple times per test. 

> If I create a new function to only load, but not verify the Delta table, should I handle the `alt_db` or alt_name` arguments like the `verify_delta_table_loaded_to_delta` function in `test_load_to_from_delta.py` does?

Yes, I think it would make sense to support some of those same options:
- spark: SparkSession
- delta_table_name: str
- s3_bucket: str
- alt_db: str = None
- alt_name: str = None
- load_command: str = "load_table_to_delta"

> If so, should I just split the function in test_load_to_from_delta.py into load_delta_table_from_postgres and verify_delta_table_loaded_to_delta functions?

I think this could make sense. Up to you though. You may want to make `verify_delta_table_loaded_to_delta` call `load_delta_table_from_postgres`, so you don't have to refactor each individual test in `test_load_to_from_delta.py`

Reply:
Created `load_delta_table_from_postgres` function in `test_load_to_from_delta.py`, and refactored `verify_delta_table_loaded_to_delta` to call it.